src/router.ts

```diff
@@ -0,0 +1,8 @@
+import  traineeRouter  from './ controllers/trainee/index';
```

Comment:
remove index from import statement, please make sure not to repeat mistakes

Reply:
Done